src/sentry/integrations/slack/message_builder/image_block_builder.py

```diff
@@ -89,3 +98,64 @@ def _build_endpoint_regression_image_block(self) -> SlackBlock | None:
             )
             sentry_sdk.capture_exception()
             return None
+
+    def _build_function_regression_image_block(self) -> SlackBlock | None:
+        logger.info(
+            "build_function_regression_image.attempt",
+            extra={
+                "group_id": self.group.id,
+            },
+        )
+
+        try:
+            organization = self.group.organization
+            event = self.group.get_latest_event_for_environments()
+            if event is None or event.occurrence is None:
+                return None
+
+            period = get_relative_time(anchor=get_approx_start_time(self.group), relative_days=14)
+            resp = client.get(
+                auth=ApiKey(organization_id=organization.id, scope_list=["org:read"]),
+                user=None,
+                path=f"/organizations/{organization.slug}/events-stats/",
+                data={
+                    "dataset": "profileFunctions",
+                    "referrer": Referrer.API_ALERTS_CHARTCUTERIE,
+                    "project": self.group.project.id,
+                    "environment": event.get_environment().name,
+                    "start": period["start"].strftime("%Y-%m-%d %H:%M:%S"),
+                    "end": period["end"].strftime("%Y-%m-%d %H:%M:%S"),
+                    "yAxis": ["p95()"],
+                    "query": f"fingerprint:{event.occurrence.evidence_data['fingerprint']}",
+                },
+            )
+
+            evidence_data = {
+                "aggregate_range_1": event.occurrence.evidence_data["aggregate_range_1"] / 1e6,
+                "aggregate_range_2": event.occurrence.evidence_data["aggregate_range_2"] / 1e6,
```

Comment:
yeah its to convert from nanoseconds to milliseconds. The generation doesn't accept units, but I wanted to keep it here in case we add additional slack-only transformations. I'll add a comment.